src/lib/dialog/dialog-animations.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+import {
+  animate,
+  state,
+  style,
+  transition,
+  trigger,
+  AnimationTriggerMetadata,
+} from '@angular/animations';
+
+/**
+ * Animations used by MatDialog.
+ */
+export namespace MatDialogAnimations {
```

Comment:
We can't use typescript namespaces; they're forbidden inside google (has to do with interop with Closure-style javascript)

Reply:
In this case what would be the proper way to namespace them? Should we have a `MatDialogAnimations` class with static properties or just an object with some properties on it? The static class has the advantage of allowing us to make the properties readonly.

Reply:
I would just do an object that we can use typescript to make immutable